suite-native/formatters/src/components/PercentageDifferenceFormatter.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+import { Text, TextProps } from '@suite-native/atoms';
+import { useFormatters } from '@suite-common/formatters';
+
+import { FormatterProps } from '../types';
+
+type NumberValues = number;
+type PercentageDifferenceFormatterProps = FormatterProps<NumberValues> & {
+    secondValue: NumberValues;
+} & TextProps;
+
+const calculatePercentageDifference = (a: number, b: number) =>
+    Math.abs(Math.round(((a - b) / b) * 100));
+
+export const PercentageDifferenceFormatter = ({
+    value,
+    secondValue,
+    ...rest
+}: PercentageDifferenceFormatterProps) => {
+    const { SignValueFormatter } = useFormatters();
+
+    // No percentage necessary for same values
```

Comment:
I do not agree :D If numbers are same it's `0%` which is valid output and it can happen.

Reply:
https://github.com/trezor/trezor-suite/pull/7586/commits/9926190c8706ad7c65ac269c7492d71ab7c73341